src/changelog/security/_posts/2023-03-08-ssh-ciphers-update.md

```diff
@@ -0,0 +1,64 @@
+---
+modified_at:	2022-03-08 10:00:00
+title:	'SSH - Adding new crypto algorithms, Sunset obsolete ones'
+---
+
+The list of allowed algorithms of `ssh.osc-fr1.scalingo.com` and
+`ssh.osc-secnum-fr1.scalingo.com` have been updated.
```

Comment:
Shouldn't we list the usage of these SSH servers? I'm not sure it's crystal clear for everyone that it is used in case of `git push` for instance. I mean, by reading this changelog entry, I'm not sure as a customer I would understand what this changes for me.

Reply:
You're right

Reply:
no clue actually